tests/framework/db/CommandTest.php

```diff
@@ -307,7 +307,7 @@ public function testBatchInsert()
 
     public function testBatchInsertWithYield()
     {
-        if (version_compare(PHP_VERSION, '5.5', '<')) {
+        if (PHP_VERSION_ID < 50500) {
```

Comment:
Why? It's less readable.

Reply:
Optimizer friendly, in case of the constant it can drop the branch when generation opcodes. One thing I'm not sure if it should be prefixed with '\' for this or not. if I find the link where this described, I'll poke you at Twitter.

Reply:
That's actually OK and makes sense but in this case it's up to code style (performance difference doesn't matter much).